CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 The following is a curated list of changes in the Enact sandstone module, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+- `sandstone/DaySelectorDecorator` to handle `number` type properly for prop `selected`,
```

Comment:
Since `DaySelectorDecorator` is a private component, we don't use this in the change log. Instead, we could use `sandstone/DayPicker`.

```suggestion
- `sandstone/DayPicker` to handle number typed `selected` prop properly in es-ES locale
```

Reply:
I've fixed it and pushed a fix-up patch.